php_memcached_arginfo.h

```diff
@@ -1,10 +1,10 @@
 /* This is a generated file, edit the .stub.php file instead.
- * Stub hash: 3e5af769d67ce91bd713bb11b325a9ccaabbfb7a */
+ * Stub hash: 8ce11ff45ccb2b1c765e5f313305f539ca3fa4f6 */
 
 ZEND_BEGIN_ARG_INFO_EX(arginfo_class_Memcached___construct, 0, 0, 0)
-	ZEND_ARG_TYPE_INFO_WITH_DEFAULT_VALUE(0, persistent_id, IS_STRING, 1, "NULL")
```

Comment:
I can't find a definition for this macro to understand what this fourth argument is.

Reply:
It looks like it was incorrectly changed to 1 in https://github.com/php-memcached-dev/php-memcached/commit/8f3c5dc0fa773aded5d169e5c5b8eab18c8931c6#diff-25c321c509d1a97040fdc193f475a75a14a42492da840d7470bd609fbbe915ed. Probably they ran make using an old version of PHP 8.0. I generated this with 8.0.0.